source/reference/s3-api-compatibility.rst

```diff
@@ -55,13 +63,11 @@ Multipart Uploads
 - :s3-api:`UploadPart <API_UploadPart.html>`
 - :s3-api:`UploadPartCopy <API_UploadPartCopy.html>`
 
-Unsupported API Object Endpoints
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code-block:: text
+Differences from S3 APIs for Multipart Uploads
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Use `+`  instead of `~` as the header marker so that it becomes a subheading of Multipart Uploads. (An h4, I guess?)